includes/widgets/image.php

```diff
@@ -318,6 +318,56 @@ protected function _register_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
```

Comment:
It's a problematic setting. It'll almost always cause a loose of proportion and distortion of the image

Reply:
The responsive option or the option itself?

Reply:
@kobizz 

if you mean that the responsive control is problematic, then I should note that it matched all the other controls above. The **Width** and **Max Width** above are slider responsive controls .

Bu if you mean that setting fixed height to an image is problematic - then you are rights it can distort the image but this is why we have the **Object Fit**. Same applies for background images, they have background-position.